document/CoreFormat.mdk

```diff
@@ -2112,6 +2109,16 @@ if any of those strings accurately describes the construct identified by this ob
 
 If none of those strings accurately describes the construct, `kind` may contain any value specified by the analysis tool.
 
+### `parentKey` property { #logicalLocation-parentKey }
+
+If the logical location represented by the `logicalLocation` object is a nested logical location,
+then the the `logicalLocation` object shall contain a property named `parentKey` whose value is a string 
+that matches the property name of the parent logical locations's `logicalLocation` object within
+`run.logicalLocations` (&sect;[#run-logicalLocations]).
```

Comment:
can we make this sentence clearer? 'the logical location's logicalLocation object' is particularly hard to parse. can't we simply say 'the string that matches the property name of the parent 'logicalLocation' object within 'run.logicalLocations'? maybe you have an even clearer alternate?